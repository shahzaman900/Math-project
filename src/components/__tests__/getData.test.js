import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../getData';

describe('Quote', () => {
  test('renders correctly', () => {
    const component = renderer.create(<Quote />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});