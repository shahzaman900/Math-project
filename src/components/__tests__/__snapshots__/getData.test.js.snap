// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote renders correctly 1`] = `
<h1>
  Loading
</h1>
`;
